cloud-init/copr_build.py

```diff
@@ -120,9 +120,12 @@ def main(srpm, copr_conf=DEFAULT_COPR_CONF, dev=None):
 
     # 2018-03-05: adding sleep to let builds ramp up
     # this after a {'state': ['Not a valid choice.']} exception
-    time.sleep(180)
+    for timeout in range(1, 10):
```

Comment:
s/timeout/_/

You don't use timeout so pyflakes should complain about that.

Reply:
TIL that it doesn't: https://github.com/PyCQA/pyflakes/issues/285